RdkServicesTest/Source/libIBus.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "audiocapturemgr_iarm.h"
```

Comment:
audiocapture should not be here

Reply:
This is needed to implementation below, his is test header, so it can be here, this has no impact on production code